SwiftUIIntegration/Notification/NotificationPresenter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  NotificationPresenter.swift
+//  SwiftUIIntegration
+//
+//  Copyright © 2025 Zendesk. All rights reserved.
+//
+import SwiftUI
+
+/// A view that presents the messaging view to present when a message notification is tapped.
+struct NotificationPresenter: View {
```

Comment:
My suggestion is to add View to avoid associations with MVP (Model-View-Presenter)
```suggestion
struct NotificationPresenterView: View {
```